x-pack/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -657,11 +657,11 @@ export default function ({ getService }: FtrProviderContext) {
         expect(task.enabled).to.eql(true);
       });
 
-      // disable the task
-      await bulkDisable([scheduledTask.id]);
-
       await retry.try(async () => {
+        // disable the task
+        await bulkDisable([scheduledTask.id]);
         const task = await currentTask(scheduledTask.id);
+        log.debug(`bulkDisable:task(${scheduledTask.id}) enabled: ${task.enabled}`);
```

Comment:
These messages are all the same, so we won't know - if the retries do retries - where they came from, if we see multiple.  I think I'd add a suffix to the message providing some context;

```suggestion
        log.debug(`bulkDisable:task(${scheduledTask.id}) enabled: ${task.enabled}; disabling the task`);
```

And then different suffixes for the other debug messages

Reply:
There are 2 test cases ` when runSoon = true` and ` when runSoon = false`
And we run `bulkDisable` and `bulkEnable` in both and i used them as prefixes. Adding more context in the messages could be helpful but `disabling the task` suffix does not help since the prefix is already `bulkDisable`.
i am adding ` when runSoon = true/false`  suffixes to make them more explicit.